Formula/jxrlib.rb

```diff
@@ -15,15 +15,14 @@ class Jxrlib < Formula
     sha256 cellar: :any_skip_relocation, yosemite:      "0dae977caf9e34289c9dd09f7e12bdf7158ccc42d9fa2dc00b4164b82c1caf3f"
   end
 
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
```suggestion
  # Enable building with CMake. Adapted from Debian patch.
  patch do
    url "https://raw.githubusercontent.com/Gcenx/macports-wine/1b310a17497f9a49cc82789cc5afa2d22bb67c0c/graphics/jxrlib/files/0001-Add-ability-to-build-using-cmake.patch"
    sha256 "beebe13d40bc5b0ce645db26b3c8f8409952d88495bbab8bc3bebc954bdecffe"
  end
```
It might be a good idea to bump the `revision` (either add `revision 1` or increment the `revision` right after `license`) so that users who have this installed get the updated version of the formula when they do `brew upgrade`.

Reply:
Done